src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/LoggerMessageDefineAnalyzer.cs

```diff
@@ -247,29 +247,8 @@ private void AnalyzeFormatArgument(OperationAnalysisContext context, IOperation
 
             switch (argumentExpression)
             {
-                case ILiteralOperation { ConstantValue: { HasValue: false, Value: string constantValue } }:
+                case IOperation { ConstantValue: { HasValue: true, Value: string constantValue } }:
                     return constantValue;
-                case IInterpolatedStringOperation interpolated:
-                    var text = "";
-                    foreach (var interpolatedStringContent in interpolated.Parts)
-                    {
-                        if (interpolatedStringContent is IInterpolatedStringTextOperation textSyntax)
-                        {
-                            text += textSyntax.Text;
-                        }
-                        else
-                        {
-                            return null;
-                        }
-                    }
-                    return text;
-                case INameOfOperation { ConstantValue: { HasValue: true, Value: string constantValue } }:
-                    return constantValue;
-                case INameOfOperation:
-                    // return placeholder from here because actual value is not required for analysis and is hard to get
-                    return "NAMEOF";
-                case IParenthesizedOperation parenthesized:
-                    return TryGetFormatText(parenthesized.Operand);
```

Comment:
Why the parenthesized operation was removed? (Note: this IOperation applies only for VB)